contrib/ckan/meta.py

```diff
@@ -0,0 +1,147 @@
+# encoding: utf-8
+
+import datetime
```

Comment:
Lets add a big comment here why we need this file (and if it should be removed on future upgrades e.g. to 2.9).